launch/org.eclipse.cdt.launch/src/org/eclipse/cdt/launch/ui/CMainTab2.java

```diff
@@ -93,7 +93,7 @@ public class CMainTab2 extends CAbstractMainTab {
 	 */
 	protected Combo fCoreTypeCombo;
 
-	private final boolean fDontCheckProgram;
+	private boolean fDontCheckProgram;
```

Comment:
Q: since removing the "final" specifier, I think this variable should be initialised here.

Reply:
I agree. I will add it.